db/schema.rb

```diff
@@ -17,15 +17,15 @@
   enable_extension "plpgsql"
 
   create_table "addresses", force: :cascade do |t|
-    t.string   "flat",        limit: 255
-    t.string   "street",      limit: 255
-    t.string   "postal_code", limit: 255
```

Comment:
When I created and migrated my databases, these limits fell off. I'm not sure why.